src/bundle.c

```diff
@@ -723,6 +728,35 @@ static int install_bundles(struct list *bundles, struct list **subs, int current
 	to_install_files = filter_out_existing_files(to_install_files);
 	grabtime_stop(&times);
 
+	/* Step 1.5: Now that we know the bundle is valid, check if we have enough space */
+	ptr = list_head(to_install_bundles);
+	while (ptr) {
+		subman = ptr->data;
+		ptr = ptr->next;
+
+		struct bundle_result *bundle = NULL;
+		bundle = calloc(sizeof(struct bundle_result), 1);
+		/* copy relevant information over for future use */
+		strncpy(bundle->bundle_name, subman->component, BUNDLE_NAME_MAXLEN - 1);
+		bundle->topsize = subman->contentsize;
+		/* do a deep copy of the includes list */
+		bundle->includes = list_deep_clone_strs(subman->includes);
+		bundle_info = list_prepend_data(bundle_info, bundle);
+		bundle_size += calculate_size(subman->component, bundle_info, 0);
+	}
+
+	fs_free = get_available_space("/usr/");
```

Comment:
You need to preppend the path informed by --path option

Reply:
Agreed 100%, will fix.